inputsources/composite/stackoutput.ftl

```diff
@@ -0,0 +1,43 @@
+[#ftl]
+
+[#-- Get stack output --]
+[#function azure_input_composite_stackoutput_filter outputFilter]
+  [#return 
+    {
+      "Subscription" : (outputFilter.Account)!accountObject.AWSId,
+      "Region" : outputFilter.Region,
+      "DeploymentUnit" : outputFilter.DeploymentUnit
+    }
+  ]
+[/#function]
+
+[#macro azure_input_composite_stackoutput_seed]
+
+  [#local deploymentOutputs = []]
+
+  [#-- ARM Stack Output Processing --]
+  [#list commandLineOptions.Composites.StackOutputs as deploymentTemplate]
```

Comment:
They shouldn't be the input template, the file should be result from a successful deploymentTemplate rather than in input template used to deploy resources

Reply:
Thanks for reminding me.

I have updated the language used for this and previous comment [here.](https://github.com/codeontap/gen3-azure/pull/56/commits/55782f86cf3b2c4d552c8b902abb0f737f036b4e)